core/issuers.js

```diff
@@ -35,6 +65,21 @@ Description must be truthy: ${description}`;
       getBalance() {
         return mintController.getAmount(payment);
       },
+      getUse() {
```

Comment:
This is the bidirectional arrow issue, whose motivation I don't yet understand.

Reply:
A use object has a reference to either an ERTP payment or purse (let's assume it's a payment for this example.) However, the nature of depositing a payment takes the contents of that payment, removes them, and places them in the purse. Then when we go to use the use object, it has nothing to use! Therefore, we need to be able to get a new use right from the purse that we deposited the payment into, just to maintain exactly the same rights that we had before.

Reply:
This became apparent to me in this test, when Bob suddenly had no use rights and I couldn't figure out why. https://github.com/Agoric/ERTP/blob/d669aba76d5ca4a4e572f7f38ea2bc55b2a5add8/demo/gallery/vat-bob.js#L30